Gemfile

```diff
@@ -1,46 +1,64 @@
 source 'https://rubygems.org'
 
 gem 'jdbc-mysql',      '5.1.28',   :platform => :jruby
+
+# Sphinx plugin for ActiveRecord/Rails
 # gem 'thinking-sphinx', '~> 3.1.0'
+# gem 'flying-sphinx',   '1.2.0'
+# Named scopes for PostgreSQL's full text search
+gem 'pg_search'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.1.8'
 
+# Ruby interface for PostgreSQL
+gem 'pg'
+
+# Authentication solution for Rails
 gem 'devise'
 
+# Cron jobs in Ruby
 gem 'whenever', :require => false
 
+# Deployment automation tool
 gem 'capistrano', '~> 3.2.0'
 
-
+# Pagination library
 gem 'will_paginate', '~> 3.0.5'
 
-#gem 'flying-sphinx',   '1.2.0'
-
+# Activity tracking for models
 gem 'public_activity'
 
+# jQuery Inplace-Editor
 gem 'best_in_place', :git => 'https://github.com/aaronchi/best_in_place.git'
 
+# Reputation behavior for Rails
 gem 'merit'
 
+# Facebook OAuth2
 gem 'omniauth-facebook'
 
-gem "haml"
-
-gem 'pg'
+# Templating engine for HTML
+gem 'haml'
 # Use SCSS for stylesheets
-gem 'sass-rails', '~> 4.0.3'
+gem 'sass-rails', '>= 3.2'
```

Comment:
Not actually downgraded. Version is `>=3.2` So it will use the latest version above 3.2. Actually I have set the parameters according to the `bootstrap-sass` [documentation](https://github.com/twbs/bootstrap-sass#a-ruby-on-rails).